<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserInfoDialog</class>
 <widget class="QDialog" name="AddUserInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加用户信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>420</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_ID">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="id_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_name">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="name_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_sex">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>性别</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="sex_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_age">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>年龄</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="age_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_post">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>职务</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="post_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_phone">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>联系方式</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="phone_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_passwd">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="passwd_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_role">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>权限</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="role_edit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>420</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2,2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="varietyBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
